tests/envs/test_cover.py

```diff
@@ -339,22 +314,32 @@ def test_cover_multistep_options():
         # Ungrasp
         np.array([0., 0., -0.1], dtype=np.float32),
     ]
-    make_video = False  # Can toggle to true for debugging
 
-    def policy(s: State) -> Action:
-        del s  # unused
-        return Action(action_arrs.pop(0))
+    policy = utils.action_arrs_to_policy(action_arrs)
+
+    # Here's an example of how to make a video within this test.
+    # env.reset("train", 0)
+    # monitor = utils.VideoMonitor(env.render)
+    # traj = utils.run_policy_with_simulator(policy,
+    #                                        env.simulate,
+    #                                        task.init,
+    #                                        lambda _: False,
+    #                                        max_num_steps=len(action_arrs),
+    #                                        monitor=monitor)
+    # Uncomment to save the video.
```

Comment:
do we still need this comment if we're already wrapping this whole block in a "Here's an example"?